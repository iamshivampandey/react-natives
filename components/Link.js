import {useState} from 'react';
import { View,Text } from 'react-native';

const STATUS = {
  HOVERED: 'hovered',
  NORMAL: 'normal',
};

export default function Link({page, children}) {
  const [status, setStatus] = useState(STATUS.NORMAL);

  return (
    <View>
        <Text
        className={status}
        href={page || '#'}
        >
         shivam {children} account
       </Text>
    </View>
    
  );
}