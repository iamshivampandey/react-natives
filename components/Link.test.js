
import renderer from 'react-test-renderer';
import Link from './Link';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <Link page="http://www.facebook.com">Facebook</Link>,
  );
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // expect(tree).toMatchSnapshot();

  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});