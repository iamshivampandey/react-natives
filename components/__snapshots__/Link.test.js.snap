// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<View>
  <Text
    className="normal"
    href="http://www.facebook.com"
  >
    shivam 
    Facebook
  </Text>
</View>
`;
